lib/db/dao/activity_topic_dao.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:async';
+
+import 'package:maui/app_database.dart';
+import 'package:maui/db/entity/activity_topic.dart';
+import 'package:sqflite/sqflite.dart';
+
+class ActivityTopicDao {
+  Future<ActivityTopic> getActivityTopic(String id, {Database db}) async {
```

Comment:
Since this is a join table, there could be multiple rows with the same activityId. So we should be returning a list of ActivityTopics for a given Activity. So this method should be named getActivityTopicsByActivityId or getAllByActivityId
The parameter should be named activityId since we are querying by activityId.

Reply:
Do we need this query as it is fetching only one id. I have written one more query which gets all the list.Shall i take this part out?

Reply:
If you think this query is not needed, you can remove it